build/release.mk

```diff
@@ -0,0 +1,42 @@
+BUILD_DIR    := ./bin/
+GORELEASER_VERSION := v0.155.0
+GORELEASER_BIN ?= bin/goreleaser
+
+bin:
+	@mkdir -p $(BUILD_DIR)
+
+$(GORELEASER_BIN): bin
+	@echo "=== [$(GORELEASER_BIN)] Installing goreleaser $(GORELEASER_VERSION)"
+	@(wget -qO /tmp/goreleaser.tar.gz https://github.com/goreleaser/goreleaser/releases/download/$(GORELEASER_VERSION)/goreleaser_$(OS_DOWNLOAD)_x86_64.tar.gz)
+	@(tar -xf  /tmp/goreleaser.tar.gz -C bin/)
+	@(rm -f /tmp/goreleaser.tar.gz)
```

Comment:
I'm fine with removing cmd prints (`@`) for echos and cases where sensitive info shouldn't be logged, but IMHO rest cmd should be logged just to know what's being executed, why are we excluding these?